python/cuml/cuml/experimental/accel/__init__.py

```diff
@@ -33,6 +34,7 @@ def _install_for_library(library_name):
 
 def install():
     """Enable cuML Accelerator Mode."""
+    rmm.mr.set_current_device_resource(rmm.mr.ManagedMemoryResource())
```

Comment:
Do we always want this on? Cudf pandas doesn't always enable it (and the setting looks like sometimes cudf may not support managed memory? I lack the context to understand this bit: https://github.com/rapidsai/cudf/blob/d4c62efb3d326a71c1cd2504ee46817ba2d3975f/python/cudf/cudf/pandas/__init__.py#L49-L51)